### PR TITLE
[Filter/Decoder] function to get property

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Fix warning on gst-inspect cmd,
1. add config-path to read plugin properties.
2. update invalid description.

### CODE REVIEW

**Review by wooksong — APPROVED** [review]
LGTM

File: gst/nnstreamer/elements/gsttensor_decoder.c [hunk 1]
```diff
@@ -515,8 +516,9 @@ gst_tensordec_set_property (GObject * object, guint prop_id,
     }
     case PROP_CONFIG:
     {
-      const gchar *config_path = g_value_get_string (value);
-      gst_tensor_parse_config_file (config_path, object);
+      g_free (self->config_path);
+      self->config_path = g_strdup (g_value_get_string (value));
+      gst_tensor_parse_config_file (self->config_path, object);
```
Comment (anyj0527) [inline gst/nnstreamer/elements/gsttensor_decoder.c:521]: May check success/failure of `gst_tensor_parse_config_file`, changing return value of the method.

Comment (jaeyun-jung) [reply gst/nnstreamer/elements/gsttensor_decoder.c:521]: it returns nothing. (void, not bool)

**Review by songgot — APPROVED** [review]
LGTM

**Review by anyj0527 — APPROVED** [review]

**Pull request merged**